corehq/apps/userreports/reports/view.py

```diff
@@ -321,3 +337,26 @@ def excel_response(self):
         file = StringIO()
         export_from_tables(self.export_table, file, Format.XLS_2007)
         return file
+
+
+class CustomConfigurableReportDispatcher(ReportDispatcher):
+    slug = prefix = 'custom_configurable'
+    map_name = 'CUSTOM_UCR'
+
+    def dispatch(self, request, *args, **kwargs):
+        domain = kwargs.get('domain')
+        report_config_id = kwargs.get('report_config_id')
+        class_path = CustomReportConfiguration.report_class_by_domain_and_id(
+            domain, report_config_id
+        )
+        report_class = to_function(class_path)
```

Comment:
it's not the fact that you store a string instead of a class that would bother me, it's the fact that you have a hidden dependency on a class that lives only in the database.

can you instead just do something similar to the custom plugins in `settings.py`?

```python
CUSTOM_UCR_EXPRESSIONS = [
    ('abt_supervisor', 'custom.abt.reports.expressions.abt_supervisor_expression'),
...
]
```